base/abstractarray.jl

```diff
@@ -1632,6 +1632,13 @@ end
 
 typed_hcat(::Type{T}, A::AbstractVecOrMat...) where {T} = _typed_hcat(T, A)
 
+# Catch indexing errors like v[i +1] (instead of v[i+1] or v[i + 1]), where indexing is interpreted as typed a comprehension. (issue #49676)
```

Comment:
```suggestion
# Catch indexing errors like v[i +1] (instead of v[i+1] or v[i + 1]), where indexing
# is interpreted as typed a comprehension. (issue #49676)
```
Line width here too (sorry I missed this before)

Reply:
Fixed. No worries I don't think I can judge, I probably hit every newbie mistake I could have made 😅.